src/haddock_benchmark_tools/__init__.py

```diff
@@ -2,10 +2,13 @@
 import logging
 import sys
 import time
+from .version import VERSION
 
-from modules.configuration import ConfigFile  # type: ignore
-from modules.haddock import HaddockWrapper, HaddockJob  # type: ignore
-from modules.dataset import Dataset  # type: ignore
+# from functools import partial
+
+from .modules.configuration import ConfigFile  # type: ignore
```

Comment:
Since we are creating a installable python package the imports should be global and not local, this also avoids lots of errors:

```python
from haddock_benchmark_tools.modules.configuration import ConfigFile
# etc
```